halo/app/app.go

```diff
@@ -78,6 +81,7 @@ type App struct {
 	EVMEngKeeper          *evmengkeeper.Keeper
 	AttestKeeper          *attestkeeper.Keeper
 	ValSyncKeeper         *valsynckeeper.Keeper
+	StakingEventKeeper    *evmstaking2.Keeper
```

Comment:
Suggest naimng it `EVMStakingKeeper` to align with `evmstaking2.Keeper` and with other module names